lang/main.json

```diff
@@ -668,6 +668,7 @@
             "e2ee": "End-to-End Encryption",
             "feedback": "Leave feedback",
             "fullScreen": "Toggle full screen",
+            "grantModerator": "Grant Moderator",
```

Comment:
Not something important and maybe out of the scope of this PR...
But I'm not sure why we need two identical strings for translation, isn't just one enough?

Reply:
It looks like it's future proofing, I see a lot of other components have this pattern where every text gets its own key, even though they might be the same in value now